### PR TITLE
github issue templates added

### PR DESCRIPTION
**Pull request by Triplkrypl**
http://redmine.havelholding.cz/issues/1951
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

### CODE REVIEW

**Comment by AlexKratky** [comment]
I have marked a few grammatical mistakes, but otherwise you did a great job :) I recommend for this Grammarly app, which can correct a lot of mistakes

**Comment by Triplkrypl** [comment]
App with grammatical check is nice, i have in browser and PhpStorm some plugins but from experience it looks than the plugin only checks words separately and not full sentence.
Can Grammarly run with PhpStorm?

**Comment by AlexKratky** [comment]
Not sure, I use VS code and there is unofficial suport, but idk about storm

File: .github/ISSUE_TEMPLATE/apiary.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Apiary
+description: Issue about documentation for ThePay.cz WS API
+labels: ["Apiary"]
+body:
+  - type: markdown
+    attributes:
+      value: "If you do not undestand something in ThePay WS API documentation, are you missing some information, or do you find the documentation inconsistent with values returned from the API, let us know in the description below."
```
Comment (joseftraxler) [inline .github/ISSUE_TEMPLATE/apiary.yml]: typo: If you don't understand something in ThePay WS API documentation, miss some information, or find the documentation inconsistent with values returned values from the API, let us know in the description below.

**Review by joseftraxler — APPROVED** [review]
great 👍

File: .github/ISSUE_TEMPLATE/apiary.yml (hunk 1 above)
Comment (Triplkrypl) [reply .github/ISSUE_TEMPLATE/apiary.yml]: Typo fixed, I looked in other templates and PhpStorm spellcheck not found any other typos like this.

Comment (joseftraxler) [reply .github/ISSUE_TEMPLATE/apiary.yml]: There is still just a little mistake: `values returned values` fix to: `returned values`, sorry.

Comment (Triplkrypl) [reply .github/ISSUE_TEMPLATE/apiary.yml]: np

Comment (Triplkrypl) [reply .github/ISSUE_TEMPLATE/apiary.yml]: fixed

**Review by joseftraxler — APPROVED** [review]

**Review by tymajiri — APPROVED** [review]

**Pull request merged**